drivers/SmartThings/zigbee-thermostat/profiles/thermostat-popp-danfoss.yml

```diff
@@ -20,4 +24,4 @@ components:
   - name: Thermostat
 metadata:
   mnmn: SmartThingsEdge
-  vid: SmartThings-smartthings-Danfoss_Ally_Radiator_Thermostat
\ No newline at end of file
+  vid: SmartThings-smartthings-Danfoss_Ally_Radiator_Thermostat
```

Comment:
@ctowns Will we be able to remove this `mnmn` and `vid` because we're adding the `heatingSetpoint` config?

Reply:
I am unsure what the `mnmn` and `vid` are, but it seems like some kind of meta data. I do not see it in another of the other patches I have with embedded configs, but I will defer to @greens on whether or not we can remove it since I am unfamiliar.

Reply:
@ctowns Since device profiles can be customized then `metadata` section might be removed.